toolkits/global/packages/global-forms/view/inputCommonAttributes.hbs

```diff
@@ -0,0 +1,18 @@
+{{#if id}} id="{{id}}"{{/if}}
+{{#if name}} name="{{name}}"{{/if}}
+{{#if value}} value="{{value}}"{{/if}}
+{{#if error}} invalid aria-invalid="true"{{/if}}
+{{#if error}} aria-describedby="error-{{id}}"{{/if}}
```

Comment:
Should these be combined into a single `if` statement?

Reply:
Maybe? I but them on separate line for readability?

Reply:
I'm a fan of clarity over brevity to help folks understand what the code is doing. 

I think keeping them as two would help :)